src/server/main_service.h

```diff
@@ -118,6 +118,8 @@ class Service : public facade::ServiceInterface {
   // Return false if command is invalid and reply with error.
   bool VerifyCommand(const CommandId* cid, CmdArgList args, ConnectionContext* cntx);
 
+  bool CheckKeysOwnership(const CommandId* cid, CmdArgList args, ConnectionContext* dfly_cntx);
```

Comment:
Please add some documentation. Please include that this function returns `true` if all keys are owned by this instance, `false` otherwise. If `false` is returned, an error has already been send to the client.